### ISSUE TITLE
raise sensical exception when LookML object instantiation fails

### ISSUE THREAD

**Issue by ryantuck** [WarbyParker/lookmlint#61]
Currently, if an invalid source view name is found within an explore within a model, we will get a `StopIteration` error thrown when attempting to instantiate the `LookML` object since a corresponding view can't be found in the project. 

A more sensical exception should be raised so the operator can figure out where the issues are.

Adding some verbosity flag `-vvv` to the CLI might also prove useful.

In the `examples/sample_repo/test.model.lkml` file, if I change the source view name from one that exists to one that doesn't like so:

```
• ~/src/lookmlint/examples/sample_repo $$$ git diff .
diff --git a/examples/sample_repo/test.model.lkml b/examples/sample_repo/test.model.lkml
index b532971..be52dbd 100644
--- a/examples/sample_repo/test.model.lkml
+++ b/examples/sample_repo/test.model.lkml
@@ -25,7 +25,7 @@ explore: orders {
 explore: inventory_transfers {

   join: source_location {
-    from: inventory_locations
+    from: inventory_locationssss
     type: inner
     relationship: one_to_one
     sql_on: ${source_location.id} = ${inventory_transfers.source_location_id} ;;
```

The linter bombs out:

```
• ~/src/lookmlint/examples/sample_repo $$$ lookmlint lint .
Model: Model(name='test')
  Explore: Explore(label=None, model='test', name='orders')
    View: orders
    View: order_items
    View: products
  Explore: Explore(label=None, model='test', name='inventory_transfers')
    View: inventory_transfers
    View: inventory_locationssss
Traceback (most recent call last):
  File "/usr/local/bin/lookmlint", line 11, in <module>
    load_entry_point('lookmlint', 'console_scripts', 'lookmlint')()
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/ryan.tuck/src/lookmlint/lookmlint/cli.py", line 138, in lint
    lkml = lookmlint.lookml_from_repo_path(repo_path)
  File "/Users/ryan.tuck/src/lookmlint/lookmlint/lookmlint.py", line 332, in lookml_from_repo_path
    lkml = LookML('/tmp/lookmlint.json')
  File "<attrs generated init 7017bdbf4b0956c1ad307a2a732246feaaffcbe1>", line 3, in __init__
  File "/Users/ryan.tuck/src/lookmlint/lookmlint/lookmlint.py", line 266, in __attrs_post_init__
    v for v in self.views if v.name == ev.source_view_name()
StopIteration
```

**Comment by weightwatchers-carlanderson** [comment]
@ryantuck yep, I ran into this too

**Comment by msandt3** [comment]
👍 for this as well
